packages/block-editor/src/hooks/use-zoom-out.js

```diff
@@ -8,46 +8,47 @@ import { useEffect, useRef } from '@wordpress/element';
  * Internal dependencies
  */
 import { store as blockEditorStore } from '../store';
+import { unlock } from '../lock-unlock';
 
 /**
- * A hook used to set the editor mode to zoomed out mode, invoking the hook sets the mode.
+ * A hook used to set the zoomed out view, invoking the hook sets the mode.
  *
- * @param {boolean} zoomOut If we should enter into zoomOut mode or not
+ * @param {boolean} zoomOut If we should zoom out or not.
  */
 export function useZoomOut( zoomOut = true ) {
```

Comment:
For me this is an example of a premature abstraction. A public API was created for a "mode" which was not yet considered stable (as indicated by the fact that the selector/action for the modes are `__unstable`.

We now have to maintain this API but make it work with the new concept which is not ideal.